### PR TITLE
[Souliss] Fix t14 typical

### PR DESCRIPTION
**Pull request by lucacalcaterra**
Fix t14 (item type must be of Switch instead of Trigger)

### CODE REVIEW

File: bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -1057,7 +1057,7 @@
 
 
 	<channel-type id="pulse">
-		<kind>trigger</kind>
+        <item-type>Switch</item-type>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml:1060]: You need to apply spotless: mvn spotless:apply

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
This PR looks terribly strange. 
For example, all your changes in SoulissGatewayDiscovery are in fact already present in repo. 
Can you provide a new PR based on current main branch with only what you would like to fix?

**Comment by lucacalcaterra** [comment]
> This PR looks terribly strange. For example, all your changes in SoulissGatewayDiscovery are in fact already present in repo. Can you provide a new PR based on current main branch with only what you would like to fix?

I created a PR to openhab:main from lucacalcaterra:souliss (the branch i worked from begin)... probably my error is this... should i work on main? Or i can continue to work on my souliss branch ?

**Comment by lolodomo** [comment]
Update your main branch and then create a new branch from your main branch. Apply your changes to this new branch and finally push your PR.

**Comment by lucacalcaterra** [comment]
I know, in fact I'm doing it ..
however it was a curiosity, I wanted to understand if I could make a pr from the old branch

**Pull request closed without merging**